Quantum-Control-Applications/Superconducting/Single Fixed Transmon/Use Case 3 - DRAG calibration/README.md

```diff
@@ -0,0 +1,75 @@
+# Derivative Removal by Adiabatic Gate (DRAG) calibration
+
+_Author: Kevin Villegas Rosales_
+
+_Important note: The code in this folder is the exact code that we used
+to run the experiment, so it is tailored for the specific software environment
+and set up. Thus, this is code is for inspirational purposes._
+
+## The goal
+
+The goal of this experiment is to calibrate the DRAG coefficient
+to increase the single qubit gate fidelities as measured with 
+randomized benchmarking (RB).
+
+## The device
+
+The chip contains 5 2D-transoms, where some of them have nearest-neighbor connectivity, 
+with flux tunable lines. For this experiment, we focus only in one transmon with a drive
+to its flux line to put the transmon  in the **sweep-spot**. All other transmons are left
```

Comment:
Double spaces

Reply:
what do you mean?

Reply:
I mean that there are two spaces between transmon and in the sweet spot

Reply:
thanks